cmd/sealer/cmd/alpha/gen.go

```diff
@@ -43,7 +43,7 @@ Then you can use any sealer command to manage the cluster like:
 
 var exampleForGenCmd = `The following command will generate Clusterfile used by sealer under user home dir:
 
-  sealer alpha gen --passwd Sealer123 --image kubernetes:v1.19.8
+  sealer alpha gen --passwd 'Sealer123' --image kubernetes:v1.19.8
```

Comment:
`--image kubernetes:v1.19.8` replace to the latest sealer image version Or give a sample .

Reply:
done